### ISSUE TITLE
Close issues that have been already reported and resolved 

### ISSUE THREAD

**Issue by goanpeca** [spyder-ide/spyder-bot#2]

**Comment by bcolsen** [comment]
Like these?

https://github.com/spyder-ide/spyder/search?q=%22ValueError%3A+cannot+set+WRITEABLE+flag+to+True+of+this+array%22&unscoped_q=%22ValueError%3A+cannot+set+WRITEABLE+flag+to+True+of+this+array%22&type=Issues

**Comment by goanpeca** [comment]
Yes @bcolsen. Perfect example! I can use that to test the bot :-). Thanks a lot

**Comment by ccordoba12** [comment]
This is another one that is reported a lot:

https://github.com/spyder-ide/spyder/issues/9147

**Comment by ccordoba12** [comment]
And another one:

https://github.com/spyder-ide/spyder/issues/9146
